deploy/base1ns-gotemplate/ns_dev.yaml

```
@@ -0,0 +1,284 @@
+          defaultRequest:
+            cpu: 10m
+            memory: 64Mi
+          type: Container
+  - apiVersion: networking.k8s.io/v1
+    kind: NetworkPolicy
+    metadata:
+      name: allow-same-namespace
+      namespace: '{{.SPACE_NAME}}-dev'
+    spec:
+      ingress:
+        - from:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)